src/emulator/auth/operations.ts

```diff
@@ -229,6 +230,33 @@ function lookup(
   };
 }
 
+function batchGet(
+  state: ProjectState,
+  reqBody: unknown,
+  ctx: ExegesisContext
+): Schemas["GoogleCloudIdentitytoolkitV1DownloadAccountResponse"] {
+  const limit = Math.min(parseInt(ctx.params.query.maxResults) || 20, 1000);
```

Comment:
Don't use `parseInt()` for base-10 parsing. Use `Number()` instead

go/tsstyle#type-coercion

Reply:
Like this? But how can we coerce it to an integer then? The code below assumes it's an integer (e.g. `array.length`)
```suggestion
  const limit = Math.min(Number(ctx.params.query.maxResults) || 20, 1000);
```

Reply:
Yes like this. Do `Math.floor(Number(...))`

Reply:
Did `Math.floor` only since the param is already converted to a number by exegeisis.